Updated the Google oath Guide (#136)
teamhanko/docs: guides/authentication-methods/oauth/google.mdx

<Step title="Obtain Google OAuth Credentials">

- To start out lets get the redirect url from the [Hanko console](https://cloud.hanko.io/), We will need this in a moment.

>   To do so, sign in to hanko and select your project. Navigate to 'Settings' in the left sidebar and then to 'Social connections'.

Reviewer: ```suggestion
  To do so, sign in to the Hanko console and select your project. Navigate to 'Settings' in the left sidebar and then to 'Social connections'.
```

>   Copy the Redirect URL.

Reviewer: ```suggestion
  Copy the 'Redirect URL'.
```

      width={500}
    />

> - Configure your Google Auth Platform, press on Get started to set it up.

Reviewer: ```suggestion
- Configure your Google Auth Platform, press on 'Get started' to set it up.
```

    <img
      className="border-[1px] border-black rounded-[10px]"
      src="/images/oauth-providers/google/googleAuthConfig.png"
      alt="Google auth configuration"
      width={500}
    />

> - Name your App and select a User support Email.

Reviewer: ```suggestion
- Name your app and select a user support Email.
```

    <img
      className="border-[1px] border-black rounded-[10px]"
      src="/images/oauth-providers/google/configurationOne.png"
      alt="Google App Information"
      width={500}
    />

- Select 'User Type' depending on your requirements
    <img
      className="border-[1px] border-black rounded-[10px]"
      src="/images/oauth-providers/google/configurationTwo.png"
      alt="Google Project Audience"
      width={500}
    />

> - Go to Google Auth Platform > AudiencePublish and your app or add test users.

Reviewer: ```suggestion
- Go to 'Google Auth Platform' > 'Audience' and publish your app.
```

      width={500}
    />

> - Select your name and paste the previous Redirect URL (that you got from the hanko console), as an Authorized redirect URL.

Reviewer: ```suggestion
- Choose a name and paste the previous Redirect URL (that you got from the hanko console), as an 'Authorized redirect URI'.
```
